test3.tf

```diff
@@ -0,0 +1,131 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "~>2.0"
+    }
+  }
+}
+provider "azurerm" {
+  features {}
+}
+
+resource "random_integer" "rnd_int" {
+  min     = 1
+  max     = 10000
+}
+
+resource random_string "password" {
+  length      = 16
+  special     = false
+  min_lower   = 1
+  min_numeric = 1
+  min_upper   = 1
+}
+
+resource "azurerm_sql_firewall_rule" "example" {
+  name                = "firewall-rule"
+  resource_group_name = "testrg"
+  server_name         = azurerm_sql_server.example.name
+  start_ip_address    = "10.0.17.62"
+  end_ip_address      = "10.0.17.62"
+}
+
+resource "azurerm_storage_account" "example" {
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure default network access rule for Storage Accounts is set to deny'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

 <!-- b0399ecc-71c6-4dc7-9a84-71f5c9742789 -->